src/test/java/it/StandaloneIT.java

```diff
@@ -45,8 +47,20 @@ public class StandaloneIT extends Base {
   private static final SubscriptionName sinkSubscriptionName =
       SubscriptionName.of(projectId, sinkSubscriptionId);
 
+  private static void requireEnvVar(String varName) {
+    assertNotNull(
+        "Environment variable " + varName + " is required to perform these tests.",
+        System.getenv(varName));
+  }
+
   @Rule public Timeout globalTimeout = Timeout.seconds(600); // 10 minute timeout
 
+  @BeforeClass
+  public static void checkRequirements() {
+    requireEnvVar("GOOGLE_CLOUD_PROJECT_NUMBER");
+    requireEnvVar("BUCKET_NAME");
```

Comment:
Should we also check GOOGLE_CLOUD_PROJECT?

Reply:
do you use that env var?